auth-web/src/components/auth/manage-business/AccountAuthorizationRequest.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="pa-5">
+    <div v-if="isLoading" class="loading-inner-container">
+      <v-progress-circular size="50" width="5" color="primary" :indeterminate="isLoading"/>
+    </div>
+
+    <div v-else-if="!isLoading && accounts && accounts.length > 0">
+      <span>Select the account you want to perform Registries activities for <strong>{{ businessName }}</strong></span>
+      <v-select
+        id="account-authorization-request-request-account-select"
+        filled req persistent-hint validate-on-blur
+        label="Select authorizing account"
+        aria-label="Select Authorizing account"
+        v-model="selectedAccount"
+        :disabled="accounts.length < 2"
+        class="business-identifier mb-n2"
+        :items="accounts"
+        item-text="name"
+        item-value="uuid"
+        @change="emitSelected(selectedAccount)"
+      />
+      <span>You can add a message that will be included as part of your authorization request. </span>
+      <v-textarea
+        class="mb-n2"
+        filled
+        id="account-authorization-request-additional-message-textarea"
+        label="Request access additional message"
+        aria-label="Request access additional message"
+        v-model.trim="requestAccessMessage"
+        @change="$emit('changeRequestAccessMessage', requestAccessMessage)"
+        maxlength="400"
+        counter="400"
+      />
+    </div>
+    <div v-else>
+      <h3 class="text-center">No authorizing accounts found</h3>
+    </div>
+  </div>
+</template>
+
+<script lang='ts'>
+import { defineComponent, ref } from '@vue/composition-api'
+import OrgService from '@/services/org.services'
+
+export default defineComponent({
+  name: 'AccountAuthorizationRequest',
+  emits: ['changeRequestAccessMessage', 'selectAccount'],
+  methods: {
+    emitSelected: function (uuid) {
+      const selectedAcc = this.accounts.find(acc => acc.uuid === uuid)
+      this.emitSelectedAccount(selectedAcc)
+    },
+    emitSelectedAccount: function (selectedAcc) {
+      this.$emit('selectAccount', selectedAcc)
+    }
+
+  },
+  props: {
+    businessName: {
+      type: String,
+      required: true
+    },
+    businessIdentifier: {
+      type: String,
+      required: true
+    }
+  },
+  setup () {
+    const accounts = ref([])
```

Comment:
Use reactive if possible here, gets rid of .value and is a bit cleaner

Reply:
Is it OK if I stick with refs here ?

What I know reactive can only be used on objects, and here I am declaring additionally string and bool which are not objects for which reactive wont work.

I think mixing them might confuse the reader what is reactive and what not, as some will have '.value' (bool and string) while the objects wont.

If it is still recommended, I will update it for sure. 

Thanks!

Reply:
Yeah use reactive, build an object

```
const person = reactive({
  name: 'Albert',
  age: 30,
  isNinja: true,
});
```

Reply:
Anyways I'll leave it up to you

Reply:
you can also return `{ ...toRefs(state) }` so that you don't need to reference 'state' in the html

Reply:
not a big deal though I think. In vue3 I generally just declare them as refs since you don't need to return each variable individually when using script setup

Reply:
can you type the accounts array if possible though?

Reply:
Updated to have single reactive object (componentState). Let me know if this is not what you had in mind. 

Just as a footnote, ref is actually recommended way to declare a reactive vars, if we want to revise this approach in the future. 
https://vuejs.org/guide/essentials/reactivity-fundamentals.html
I did not know that before today though :D 

Thanks!

Reply:
Should I revert it back maybe ? I do not mind doing that as ref() looks nicer for me

Reply:
I'll leave it with reactive for now. Let me know if there are any changes around the componentState variable I added need to be changed/updated